tests/e2e/configurer/chain/commands.go

```diff
@@ -35,10 +38,40 @@ func (n *NodeConfig) CreatePool(poolFile, from string) uint64 {
 	err = util.Cdc.UnmarshalJSON(bz, &numPools)
 	require.NoError(n.t, err)
 	poolID := numPools.NumPools
-	n.LogActionF("successfully created pool %d", poolID)
+	n.LogActionF("successfully created balancer pool %d", poolID)
 	return poolID
 }
 
+func (n *NodeConfig) CreateConcentratedPool(from string) poolmanagertypes.PoolI {
+	n.LogActionF("creating concentrated pool")
+
+	var (
+		denom1             = "uosmo"
+		denom2             = "gamm/pool/1"
+		tickSpacing        = 1
+		exponentAtPriceOne = -1
+	)
+
+	cmd := []string{"osmosisd", "tx", "concentratedliquidity", "create-concentrated-pool", denom1, denom2, fmt.Sprintf("%d", tickSpacing), fmt.Sprintf("[%d]", exponentAtPriceOne), fmt.Sprintf("--from=%s", from)}
+	_, _, err := n.containerManager.ExecTxCmd(n.t, n.chainId, n.Name, cmd)
+	require.NoError(n.t, err)
+
+	path := "/osmosis/concentratedliquidity/v1beta1/pools"
+
+	bz, err := n.QueryGRPCGateway(path)
+	require.NoError(n.t, err)
+
+	var pools cltypes.QueryPoolsResponse
+	err = util.Cdc.UnmarshalJSON(bz, &pools)
+	require.NoError(n.t, err)
+	n.LogActionF("successfully created concentrated pool")
+
+	var createdPool poolmanagertypes.PoolI
+	anyPool := pools.Pools[len(pools.Pools)-1]
+	err = util.Cdc.UnpackAny(anyPool, &createdPool)
+	require.NoError(n.t, err)
+	return createdPool
```

Comment:
Let's make this a similar query method in `queries.go` please

Let `CreateConcentratedPool` be returning pool id only

Reply:
Please make that query return a `ConcentratedPoolExtension` instead of `PoolI`. Please perform a type cast to achieve that